.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "plugins/wasmedge_rwkv/ggml"]
```

Comment:
We don't accept submodules; please follow the same procedure for handling dependencies.